src/main/java/baseball/GameController.java

```diff
@@ -0,0 +1,67 @@
+package baseball;
+
+import static baseball.message.SystemMessage.*;
+
+import baseball.domain.Computer;
+import baseball.domain.Player;
+import baseball.domain.Referee;
+
+import camp.nextstep.edu.missionutils.Console;
+
+public class GameController {
+	public static final String RESTART_COMMAND = "1";
+	public static final String END_COMMAND = "2";
+
+	private final Player player;
+	private final Computer computer;
+	private final Referee referee;
+
+	public GameController() {
+		player = new Player();
+		computer = new Computer();
+		referee = new Referee();
+	}
```

Comment:
객체 사용에 필요한 변수를 자체적으로 내부에서 생성 중인데요.
컨트롤러라 크게 외부 주입이 필요하지 않더라도 외부에서 인자를 받아 주입하도록 코드를 작성하도록 노력해보세요.  A 객체가 B, C, D 객체를 의존할 때 외부에서 주입받는 방식과 내부에서 자체 생성하는 방식은 어떤 차이가 있고 외부 주입 방식에는 어떤 장점이 존재할지 고민해봐도 좋겠네요.

객체지향과 매우 밀접한 부분입니다 :)

Reply:
```suggestion
	public GameController(Player player, Computer computer, Referee referee) {
		this.player = player;
		this.computer = computer;
		this.referee = referee;
	}
```

```
// Application.java
public class Application {
	public static void main(String[] args) {
		Player player = new Player();
		Computer computer = new Computer();
		Referee referee = new Referee();
		GameController baseBall = new GameController(player, computer, referee);
		baseBall.run();
	}
}
```

이런식으로 하면 외부에서 주입되는 것이 맞을까요?? 
외부에서 주입됨으로써 더욱 유연해져 변경에 용이할 듯 합니다!